OpenSSL/test/util.py

```diff
@@ -119,7 +121,8 @@ def format_leak(p):
 
             # Figure the last interesting frame will always be CRYPTO_malloc,
             # since that's where we hooked in to things.
-            while c_stack and 'CRYPTO_malloc' not in c_stack[0] and 'CRYPTO_realloc' not in c_stack[0]:
+            while c_stack and 'CRYPTO_malloc' not in c_stack[0] and \
+                    'CRYPTO_realloc' not in c_stack[0]:
```

Comment:
Can you use parenthesis here, we've avoided using `\` line continuations everywhere else.